kernel/lockfs.c

```diff
@@ -713,6 +731,12 @@ int myst_lockfs_init(myst_fs_t* fs, myst_fs_t** lockfs_out)
     lockfs->fs = fs;
     *lockfs_out = &lockfs->base;
 
+    if (!_inited)
```

Comment:
Statically defined myst mutexes don't need to be initialized (since statics are zero-initialized). We should probably add a "MYST_MUTEX_INITIALIZER" macro for clarity.

Reply:
```
#define MYST_MUTEX_INITIALIZER { MYST_SPINLOCK_INITIALIZER }
```

```
static myst_mutex_t _mutex = MYST_MUTEX_INITIALIZER ;
```

Reply:
Thanks this is really neat.